.github/workflows/deploy-backend-dev.yml

```diff
@@ -23,8 +23,12 @@ jobs:
         uses: ./.github/actions/setup-repo
       - name: Build and Deploy to Firebase
         uses: w9jds/firebase-action@v13.0.2
+      - name: Set Admin Role
+        run: yarn  firebase-admin setRole --email="${{ secrets.TEST_ADMIN_USERNAME }}" --role="admin"
```

Comment:
Could we just run `setRole` ourselves once to make this account an admin and then leave that account as an admin on dev? It seems like once this is setup, we don't need to fix it on subsequent deploys.